client/structs/csi.go

```diff
@@ -75,3 +75,28 @@ type ClientCSIControllerAttachVolumeResponse struct {
 	// subsequent `NodeStageVolume` or `NodePublishVolume` calls
 	PublishContext map[string]string
```

Comment:
@endocrimes it occurs to me we need to add field to `nomad/structs/csi.go#CSIVolumeClaimResponse` for the clients to consume as well. I'll push that up as a separate PR shortly.